test/org/sagebionetworks/bridge/validators/StudyValidatorTest.java

```diff
@@ -210,5 +214,31 @@ public void cannotSetMaxNumOfParticipantsLessThanZero() {
         study.setMaxNumOfParticipants(-100);
         assertCorrectMessage(study, "maxNumOfParticipants", "maxNumOfParticipants must be zero (no limit on enrollees) or higher");
     }
+    
+    @Test
+    public void shortListOfDataGroupsOK() {
+        study.setDataGroups(Sets.newHashSet("beta_users", "production_users", "testers", "internal"));
+        Validate.entityThrowingException(StudyValidator.INSTANCE, study);
+    }
+    
+    @Test
+    public void longListOfDataGroupsInvalid() {
+        study.setDataGroups(newLinkedHashSet("Antwerp", "Ghent", "Charleroi", "Liege", "Brussels-City", "Bruges", "Schaerbeek", "Anderlecht", "Namur", "Leuven", "Mons", "Molenbeek-Saint-Jean"));
+        assertCorrectMessage(study, "dataGroups", "dataGroups will not export to Synapse (string is over 100 characters: 'Antwerp, Ghent, Charleroi, Liege, Brussels-City, Bruges, Schaerbeek, Anderlecht, Namur, Leuven, Mons, Molenbeek-Saint-Jean')");
+    }
```

Comment:
Might be easier if we move MAX_SYNAPSE_LENGTH to a member method, then mock (spy) it out to a smaller value, so our tests are less cumbersome.